<launch>
  <node pkg="avoid_obstacles" type="avoid_obs" name="avoid_obs" output="screen">
    <param name="plan_rate_hz" value="1.0" />
    <param name="map_res_m" value="1.0" />
    <param name="map_size" value="150" />
    <param name="max_range" value="40.0" />
    <param name="plan_range" value="35.0" />
    <param name="clear_decrement" value="-5" />
    <param name="fill_increment" value="20" />
    <param name="use_Astar" value="true" />
    <param name="use_PotFields" value="false" />
  </node>
</launch>
